python/samples/concepts/audio/audio_player.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import io
+import logging
+import wave
+from typing import ClassVar
+
+import pyaudio
+
+from semantic_kernel.contents.audio_content import AudioContent
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+
+logging.basicConfig(level=logging.WARNING)
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+class AudioPlayer(KernelBaseModel):
```

Comment:
when a class is not related to SK but just used in a sample I always prefer to just import BaseModel directly from pydantic rather then KernelBaseModel